synctl

```diff
@@ -2484,6 +2542,18 @@ class ParseParameter:
             '--timeout', type=str, help='set timeout, accept <number>(ms|s|m)')
         patch_exclusive_group.add_argument(
             '--script-file', type=str, help="script file name")
+        patch_exclusive_group.add_argument(
+            '--entry-file', type=str, help="script file name")
```

Comment:
The help message is same with `--script-file`, maybe `entry file of a bundle test`.